doc/source/whatsnew/v1.4.0.rst

```diff
@@ -603,6 +603,7 @@ Other Deprecations
 - Deprecated passing non boolean argument to sort in :func:`concat` (:issue:`41518`)
 - Deprecated passing arguments as positional for :func:`read_fwf` other than ``filepath_or_buffer`` (:issue:`41485`):
 - Deprecated passing ``skipna=None`` for :meth:`DataFrame.mad` and :meth:`Series.mad`, pass ``skipna=True`` instead (:issue:`44580`)
+- Deprecated the behavior of :func:`to_datetime` with the string "now" with ``utc=False``; in a future version this will match ``Timestamp("now")``, which in turn matches :meth:`Timestamp.now` (:issue:`18705`)
```

Comment:
Maybe good to mention here that `Timestamp("now")`/`Timestamp.now` returns local time.